Ry4

ryhorowitz opened the pull request:
## Description
Map skeleton. Just a map with a marker. 

If anyone can help me with Google API calls that would be most appreciated.  It's been 5 days now and I got nothing.

## Issue tracking
<!-- Link to Trello ticket -->

shabbyblue16 committed:
Added async selector in client to get data from db

shabbyblue16 committed:
Added expo publish, qr code commented into PRs, and dependency caching for the yml script

shabbyblue16 committed:
Merge branch 'dev' of https://github.com/rpp30-boc-heartbleed/whimsical into dev

shabbyblue16 committed:
Added filterByName selector

shabbyblue16 committed:
Working on search friends functionality

shabbyblue16 committed:
Finished a really basic search function

shabbyblue16 committed:
Changed manifest name

shabbyblue16 committed:
Merge branch 'dev' of https://github.com/rpp30-boc-heartbleed/whimsical into dev

Laweeza committed:
replaced geolocation dependency with geolocation-service

shabbyblue16 committed:
Fixed app.json assets refs

shabbyblue16 committed:
Merge branch 'dev' into feature-friends

shabbyblue16 committed:
linter

shabbyblue16 committed:
save

shabbyblue16 committed:
Merge branch 'dev' of https://github.com/rpp30-boc-heartbleed/whimsical into dev

shabbyblue16 committed:
fixes

shabbyblue16 committed:
Merge branch 'dev' into feature-friends

shabbyblue16 committed:
Modularized Search

shabbyblue16 committed:
Modularized List in FriendsList

Laweeza committed:
MAP works on expo for errand tracker

Laweeza committed:
google map rendering on tracker component

OjeikuA committed:
profile built

OjeikuA committed:
profile model

OjeikuA committed:
profile controller

OjeikuA committed:
booting up to test

OjeikuA committed:
adding profile dummy data

Laweeza committed:
fleshed out errandRunner bottomsheet for map tracker

Laweeza committed:
pulled down will investigate package.json later

ryhorowitz committed:
map round 4

Laweeza committed:
working on topbar skeleton

Laweeza committed:
mvp top bar

Laweeza committed:
google map marker and directions implemented

Laweeza committed:
made stand in counter for text for now

Laweeza committed:
Merge branch 'main' into errandTracker

Laweeza committed:
Got .env to work with component

Laweeza committed:
changed eta time to be more realistic

ryhorowitz committed:
map skeleton. Cannot for the life of me figure this ish out.

Laweeza reviewed (approved):
Skeleton looks okay for component

Laweeza commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Map Skeleton](https://trello.com/c/ijYO33lC/45-map-skeleton)

Laweeza committed:
made message button smaller and implemented alert for now

Laweeza committed:
made message button smaller and implemented alert for now

Laweeza committed:
locked mvp features

Laweeza committed:
locked mvp features

ryhorowitz committed:
Added state updating when the map is moved by the user.

shabbyblue16 committed:
Added avatar and chat icons

shabbyblue16 committed:
Working on icon modal

ryhorowitz committed:
Need to find out if I can make activeAPI calls to Google Maps Librarys.Still can't figure this out.

shabbyblue16 committed:
Added subcomponents folder with index

shabbyblue16 committed:
Added toggle button

shabbyblue16 committed:
Toggle filters friends who are currently on errands

shabbyblue16 committed:
Created add friend button

Laweeza committed:
Added to Read.Me 

Added Pending Features to Read.ME
- Filled out small blurb for 'About the App' section for now

Laweeza committed:
edited

Laweeza committed:
Edited

shabbyblue16 committed:
Merge branch 'dev' of https://github.com/rpp30-boc-heartbleed/whimsical into dev

shabbyblue16 committed:
Deleted app.config.js

shabbyblue16 committed:
Merge branch 'dev' into feature-friends

shabbyblue16 committed:
Commented out cd script

shabbyblue16 commented on .gitignore:
```diff
@@ -10,6 +10,6 @@ npm-debug.*
 *.orig.*
 web-build/
-
+app.json
```
Good thought with the API keys. We'll have to make sure everybody knows to recreate app.json file locally.

shabbyblue16 commented on client/components/Map/Map.jsx:
```diff
@@ -0,0 +1,74 @@
+
+const Map = () => {
+  const currentView = useRecoilValue(currentMapView);
+  const userLocation = useRecoilValue(userLocationState);
```
I think to use this atom `useRecoilState(currentMapView)` is needed, but if this works then no big deal.

shabbyblue16 reviewed (approved):
Good stuff, Google Maps API is a beast. My component's pretty simple so I'm happy to help if you need to delegate.

Laweeza committed:
Merge pull request #18 from rpp30-boc-heartbleed/errandTracker

Errand Tracker Skeleton

OjeikuA committed:
image uploads complete

OjeikuA committed:
update password complete

OjeikuA committed:
User Profile with Edit Capabilities

OjeikuA committed:
fixed conflicts

OjeikuA committed:
resolving conflicts with dev branch

OjeikuA committed:
fixed conflicts x2

surekhaw committed:
Merge pull request #21 from rpp30-boc-heartbleed/user-profile-1

User profile 1

shabbyblue16 committed:
Merge branch 'dev' of https://github.com/rpp30-boc-heartbleed/whimsical into dev

shabbyblue16 committed:
Merge branch 'dev' into feature-friends

shabbyblue16 committed:
Merge pull request #20 from rpp30-boc-heartbleed/feature-friends

FriendsList Component

shabbyblue16 committed:
Merge branch 'ry4' of https://github.com/rpp30-boc-heartbleed/whimsical into ry4

shabbyblue16 committed:
package-lock
